### PR TITLE
Updated `skyux new` to use Angular CLI-based projects

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**

### CODE REVIEW

File: lib/new.js [hunk 1]
```diff
@@ -104,89 +110,39 @@ async function checkoutNewBranch() {
  * Removes the .git folder. Prepares files.
  * @name cleanupTemplate
```
Comment (Blackbaud-SteveBrush) [inline lib/new.js]: Probably should rename this `@name` value, or remove it.

**Review by Blackbaud-SteveBrush — COMMENTED** [review]
All commands work great (even `skyux check-workspace`)! Some thoughts:
- When I ran `ng serve` the page was blank (but working). We should either add some basic content to the home page, or create a work item to add it later.
- The `skyux eject` command assumes the project is public (a SPA is determined to be "internal" if the git remote is hosted on ADO, or if the `--internal` flag is set). Should we also assume "public" for `skyux new`, or simply not support `skyux new` for public repos at all?

File: lib/new.js (hunk 1 above)
Comment (Blackbaud-PaulCrowder) [reply lib/new.js]: Fixed.

**Comment by Blackbaud-PaulCrowder** [comment]
> All commands work great (even `skyux check-workspace`)! Some thoughts:
> 
> * When I ran `ng serve` the page was blank (but working). We should either add some basic content to the home page, or create a work item to add it later.
> * The `skyux eject` command assumes the project is public (a SPA is determined to be "internal" if the git remote is hosted on ADO, or if the `--internal` flag is set). Should we also assume "public" for `skyux new`, or simply not support `skyux new` for public repos at all?

- There really shouldn't be any content on the page that has the router outlet. I'd prefer to let the consumer start with a clean slate for now rather than always having to go delete something.
- I'm not sure we need `skyux new` for public repos. We can talk more about that offline.

File: lib/utils/prettier-utils.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+  const packages = await Promise.all(
+    [
+      latestVersion('prettier'),
+      latestVersion('eslint-config-prettier')
+    ]
+  );
+
+  packageJson.devDependencies['prettier'] = packages[0];
+  packageJson.devDependencies['eslint-config-prettier'] = packages[1];
```
Comment (Blackbaud-SteveBrush) [inline lib/utils/prettier-utils.js:65]: We're also installing `eslint-plugin-prettier` on our builders. Should we install that with our consumers, too? 
See: https://github.com/prettier/eslint-plugin-prettier#recommended-configuration

Comment (Blackbaud-PaulCrowder) [reply lib/utils/prettier-utils.js:65]: When I was doing research on how to integrate Prettier and ESLint, I came across this guidance from Prettier which seemed to discourage the use of `eslint-plugin-prettier`. I think `eslint-config-prettier` will give us what we need without the plugin. https://prettier.io/docs/en/integrating-with-linters.html#notes

> These plugins were especially useful when Prettier was new. By running Prettier inside your linters, you didn’t have to set up any new infrastructure and you could re-use your editor integrations for the linters. But these days you can run prettier --check . and most editors have Prettier support.

Comment (Blackbaud-SteveBrush) [reply lib/utils/prettier-utils.js:65]: Good info. Thanks!

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**